build.gradle

```diff
@@ -24,12 +24,16 @@ ext {
     compileSdkVersion = 29
 
     // App dependencies
-    supportLibraryVersion = '28.0.0'
-    runtimeVersion = '100.6.0'
-    butterknifeVersion = '8.8.1'
+    appcompatLibraryVersion = '1.1.0'
```

Comment:
I would suggest moving all of your versions to a gradle file called `version.gradle` like the samples team did [here](https://github.com/Esri/arcgis-runtime-samples-android/blob/master/version.gradle) just to keep everything uniform.